include/common/s11_structs.h

```diff
@@ -12,6 +12,7 @@
 #include <stdlib.h>
 #include <arpa/inet.h>
 #include <unistd.h>
+#include <nas_structs.h>
```

Comment:
why s11 need nas_Struct ?

Reply:
structs pco and bearer_tft declared in nas_structs.h are used inside the bearer_ctxt_t here. Hence , it is required.